doc/en/reference/reference.rst

```diff
@@ -1839,9 +1839,17 @@ passed multiple times. The expected format is ``name=value``. For example::
        pytest testing doc
 
 
-.. confval:: tmp_path_retention_count
+.. confval:: collect_imported_tests
+
+   Setting this to `false` will make pytest collect classes/functions from test
+   files only if they are defined in that file (as opposed to imported there).
+
+    .. code-block:: ini
```

Comment:
RST is fidgety with spaces, this needs to be indented according to the block above otherwise it will not render correctly:

![image](https://github.com/user-attachments/assets/73de4150-60b4-49c2-8cad-f9e09ecb4dc8)

(Compare with `tmp_path_retention_count` below).

```suggestion
   .. code-block:: ini
```